bpf/kprobe_pwru.c

```diff
@@ -262,6 +270,10 @@ set_output(struct sk_buff *skb, struct event_t *event, struct config *cfg) {
 
 	if (cfg->output_skb)
 		set_skb_btf(skb, &event->print_skb_id);
+
+	if (cfg->output_stack) {
+		event->print_stack_id = bpf_get_stackid(ctx, &print_stack_map, BPF_F_REUSE_STACKID | BPF_F_FAST_STACK_CMP);
```

Comment:
I'd avoid setting `BPF_F_REUSE_STACKID`, as otherwise we risk to print unrelated stacks.

Reply:
done